WebApp/template/root.tpl

```diff
@@ -141,9 +141,28 @@
             </td>
           </tr>
         </table>
+      </div>
+  </div>
 
+  <div id="app-dirac-welcome">
+      <div class="app-dirac-welcome-text">
+        <h1>Weclome to DIRAC!</h1>
+        <b>D</b>istributed <b>I</b>nfrastructure with <b>R</b>emote <b>A</b>gent <b>C</b>ontrol INTERWARE is a software framework for distributed computing providing a complete solution to one (or more) user community requiring access to distributed resources. DIRAC builds a layer between the users and the resources offering a common interface to a number of heterogeneous providers, integrating them in a seamless manner, providing interoperability, at the same time as an optimized, transparent and reliable usage of the resources. 
+        <a href="http://diracgrid.org">Discover more...</a>
```

Comment:
Better to open the new link in a new tab. Could you please use: <a target="_blank" href="http://diracgrid.org">Discover more..</a> ?